### DISCUSSION TITLE
Is it possible to not show ... in mobile just show ... and one last li only

### DISCUSSION THREAD

**Discussion by awais627** [AdeleD/react-paginate#445] [Q&A] [answered]
In Mobile only show 1 ... not less or more and after ... show only last li only not more then 1 i want to achieve this belo
![Screenshot from 2022-08-25 20-04-01](https://user-images.githubusercontent.com/104203160/186701079-b33ef7e7-8eb0-4928-8cdc-451761f715dc.png)
w

**Comment by vladpereverzyev** [comment, marked as answer]
You can control this with the breakLabel and CSS, but react-paginate doesn't have a built-in option to show only one page after the ellipsis.

You can use CSS/media queries to hide the unwanted page items on mobile and keep only the last page visible. Alternatively, you could use pageRangeDisplayed and marginPagesDisplayed and adjust the values for the mobile layout.

For example, try reducing pageRangeDisplayed to 1 and marginPagesDisplayed to 1 on mobile.

**Reply by awais627** [reply]
@vladpereverzyev  Thank you for the answer! 😓 

I asked this in 2022 as an intern. At this point I don't even remember what I was trying to build, why I needed this pagination behavior, or what project this was for.

It’s now 2026. I’m no longer an intern, I’ve worked on countless projects, dealt with production bugs, deployments, APIs, databases, cloud infrastructure… and somehow this GitHub discussion was still waiting for its answer. 😭 Honestly, this is no longer a technical answer. This is closure.

Still, thank you sooooooo much for completing a task that has been sitting in my backlog longer than some of my actual projects. 🙏 ❤️
